en/docs/learn/consume-api/manage-application/advanced-topics/adding-an-application-key-generation-workflow.md

```diff
@@ -8,272 +8,59 @@ This section explains as to how you can attach a custom workflow to the **applic
 
 
-3.  Sign in to the API Developer Portal (<https://localhost:9443/devportal>) as a Developer Portal user and open the application with which you subscribed to the API.
-
     !!! note
         If you do not have an API already created and an Application subscribed to it, follow [Create a REST API]({{base_path}}/learn/design-api/create-api/create-a-rest-api/), [Publish an API]({{base_path}}/learn/design-api/publish-api/publish-an-api/), and [Subscribe to an API]({{base_path}}/learn/consume-api/manage-subscription/subscribe-to-an-api) to create an API and subscribe to it.
 
-
-4.  Click **Applications**, **Production Keys**, and **Generate Keys**.
+5.  Select **Production Keys** or **Sandbox Keys** from the side navbar and click on **GENERATE KEYS**.
     
-     It invokes the `ApplicationRegistrationWorkFlowProcess.bpel` that is bundled with the `ApplicationRegistrationWorkflowProcess_1.0.0.zip` file and creates a HumanTask instance that holds the execution of the BPEL process until some action is performed on it.
-
-     [ ![Generate keys for an application]({{base_path}}/assets/img/learn/add-registration-wf-generate-keys.png) ]({{base_path}}/assets/img/learn/add-registration-wf-generate-keys.png)
-
+     [ ![Generate keys for an application]({{base_path}}/assets/img/learn/application-registration-generate-keys.png) ]({{base_path}}/assets/img/learn/application-registration-generate-keys.png)
 
-     Note that a message appears saying that the request is successfully submitted when the BPEL is invoked correctly.
 
-5.  Sign in to the Admin Portal (`https://<Server Host>:9443/admin`) with admin credentials and list all the tasks for application registrations. 
+     Note that this message will appear if the application key generation workflow is correctly enabled.
 
-6. Click **Start** to start the Human Task and then change its state. 
+     [ ![Generate keys for an application message]({{base_path}}/assets/img/learn/application-key-genereation-message.png) ]({{base_path}}/assets/img/learn/application-key-genereation-message.png)
 
-7. Click **Approve** and **Complete** to complete the task.
+6.  Sign in to the Admin Portal (`https://<Server Host>:9443/admin`) with admin credentials and list all the tasks for application registrations from **Tasks** --> **Application Registration** and click on approve or reject to approve or reject the application key generation pending request. 
 
-     This resumes the BPEL process and completes the registration.
-
-    [![]({{base_path}}/assets/img/learn/add-registration-wf-approval.png)]({{base_path}}/assets/img/learn/add-registration-wf-approval.png)
+    [![]({{base_path}}/assets/img/learn/application-registration-pending-list.png)]({{base_path}}/assets/img/learn/application-registration-pending-list.png)
```

Comment:
the alt text does not need the path to be included

Reply:
removed unnecessary parts